exercises/pig-latin/src/test/java/PigLatinTranslatorTest.java

```diff
@@ -1,85 +1,95 @@
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-import java.util.Arrays;
-import java.util.Collection;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(Parameterized.class)
 public class PigLatinTranslatorTest {
 
-    private String englishPhrase;
-    private String pigLatinTranslation;
+    private PigLatinTranslator pigLatinTranslator;
 
-    @Parameterized.Parameters(name = "{index}: expected \"{0}\" to translate to the pig latin phrase \"{1}\"")
-    public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][]{
-                // Ay is added to words that start with vowels
-                {"apple", "appleay"},
-                {"ear", "earay"},
-                {"igloo", "iglooay"},
-                {"object", "objectay"},
-                {"under", "underay"},
+    public PigLatinTranslatorTest() {
```

Comment:
I think you can probably drop this constructor, as Java will automatically add a default constructor for you!

See this [stackoverflow question](https://stackoverflow.com/questions/4488716/java-default-constructor) for more info! 🙂